class URI_ERROR(Exception):
    pass
